### DISCUSSION TITLE
Move all graph displays to the left at startup

### DISCUSSION THREAD

**Discussion by jazzmonger** [dbuezas/lovelace-plotly-graph-card#332] [General]
Every time I fire up ha and look at this (and other) graphs, I have to slide them to the left a little bit to see all the values because the labels overlap them.  Is there a way to offset the display so it defaults this way?
![IMG_2594](https://github.com/dbuezas/lovelace-plotly-graph-card/assets/52110065/c51bd33a-bef3-4a3a-888c-419ec21c9b49)
![IMG_2598](https://github.com/dbuezas/lovelace-plotly-graph-card/assets/52110065/a273b3f1-e59d-405e-9540-1d9144070a81)

**Comment by dbuezas** [comment]
Yes, use `time_offset`. See here: https://github.com/dbuezas/lovelace-plotly-graph-card#show_value

**Reply by jazzmonger** [reply]
Perfect!
